src/main/70-operational-database.tf

```diff
@@ -0,0 +1,30 @@
+module "aurora_postgresql_v2" {
```

Comment:
- Needs to allow bastion host SG
- Create a separate cluster parameter group (there is a variable in this module)
- Add deletion protection